### ISSUE TITLE
Your Password does not Satisfy the Current Policy Requirements

### ISSUE THREAD

**Issue by efimov90** [dbeaver/dbeaver#13897]
#### System information: 
- Windows 10 19043.1165 21H1
- 21.2.0.202108310918

#### Describe the problem you're observing:

When you try to create user with validating password plugin (validate password as example) you get error from title.
It appears because of generated code to create user.

#### Steps to reproduce, if exist:

I think it will work at any setup with this type of plugins but in my case i use this setup:
docker container with percona/percona-server with installed validate password (as plugin or as component)

Try to create new user with password that satisfy policy, as example 'P@ssw0rd'.
In generated code first line will be create user but without password <- this cause of error.

**Comment by uslss** [comment]
@efimov90 what database do you use?

**Comment by efimov90** [comment]
percona/percona-server docker image
version 8.0.25-15
version comment Percona Server (GPL), Release 15, Revision a558ec2

**Comment by uslss** [comment]
what driver do you use?
could you provide the generated script?

**Comment by efimov90** [comment]
Driver in connection: 'MySQL'

Generated script is:
```
CREATE USER 'test'@'%';
ALTER USER 'test'@'%' IDENTIFIED BY 'P@ssw0rd';
```

But must be:
```
CREATE USER 'test'@'%' IDENTIFIED BY 'P@ssw0rd';
```

**Comment by uslss** [comment]
thanks for the report.
duplicates #11850 .
Will be fixed in 21.2.1
